docs/TrafficSplitting.md

```diff
@@ -0,0 +1,163 @@
+
+- **`--traffic-latest Percent`**
+  - `--traffic-latest` flag requires one value
+  - `Percent` integer denotes the traffic portion to assign to the latest ready Revision of the Service
+  - If the `service update` command is updating the Configuration values for the Service (along with traffic flags), this reference will be pointed to the created Revision after applying the update
+  - `-traffic-latest` flag can be specified only once in a `kn service update` command
+
+### 2. Tag Revisions:
+A Tag in traffic block of Service creates a custom URL, which points to the referenced Revision.
+A user can define a unique Tag for an available Revision of the Service thereby generating a custom URL of format `http(s)://TAG-SERVICE.DOMAIN`.
+A given tag must be unique in traffic block of the Service. `kn` supports assigning and unassigning custom tags for revisions of the Service as part of `kn service update` command as follow:
+
+- **`--tag-revision RevisionName:Tag`**
+  - `--tag-revision` flag requires two values separated by a colon
+  - `RevisionName` string refers to name of the `Revision`
+  - `Tag` string denotes the custom Tag to be given for this target
+  - `--tag-revision` flag may assign different tags to same Revision
+  - `--tag-revision` flag can be specified multiple times
+
+- **`--tag-latest Tag`**
+  - `--tag-latest` flag requires one value
+  - `Tag` string denotes the custom Tag to be given to the latest ready Revision of the Service
+  - If the `service update` command is updating the Configuration values for the Service (along with tag flags), this reference will be pointed to the created Revision after applying the update
+  - `--tag-latest` flag can be specified only once in a `kn service update` command
+
+*Note*:
+- Assigned a Tag to a particular Revision, a user can reference the Revision by its Tag in `--traffic` flag as `--traffic Tag:Percent`.
+- User can specify `--traffic` and `--tag-revision` flags in same `service update` command and allowed to use the new `Tag` for Revision reference
+  for eg: `kn service update --tag-revision echo-abcde:stable --traffic stable:100`.
+
+### 3. Untag Revisions:
+Assigned tags to revisions in traffic block can be unassigned, thereby removing any custom URLs. A user can unassign the tags for revisions as part of `kn service update` command as follow:
+
+- **`--untag-revision Tag`**
+  - `--untag-revision` flag requires one value
+  - `Tag` string denotes the unique Tag in traffic block of the Service which needs to be unassigned and thereby also removing respective custom URL
+  - `--untag-revision` flag can be specified multiple times
+
+*Note*:
+- `--untag-revision` flag's action takes first precedence, meaning if `--untag-revision` and `--tag-revision` flags specified together for revisions, all the mentioned tags will be unassigned from revisions first followed by tagging them.
```

Comment:
Split into  two sentences by putting a period after "precedence", and removing the word "meaning"